### ISSUE TITLE
Some Issues Regarding Data Preparation

### ISSUE THREAD

**Issue by jah417051695** [arvin1367/3-D-Deformation-Inversion#6]
Dear author,

Regarding the files needed in the script "3D inversion for land subsidencees":

ascending_los (dir)
descending_los (dir)
date_ascending.xls (table)
date_descending.xls (table).
I would like to ask if the los folders only need a deformation rate, or do they require cumulative deformation amounts for each moment, or each geocoded interferogram?

I look forward to your reply, thank you.

**Comment by arvin1367** [comment]
Yes, a deformation image pair with phase shift deformation is required.
